.github/workflows/update-base.yml

```diff
@@ -0,0 +1,76 @@
+name: Update base image base
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Monday at midnight
+    - cron: "0 0 * * 1"
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set git identity
+        run: |
+          git config --global user.name $GITHUB_USER
+          git config --global user.email $GITHUB_EMAIL
+        env:
+          GITHUB_USER: roboquat
+          GITHUB_EMAIL: roboquat@gitpod.io
```

Comment:
I wonder if we need to grant `roboquat` permissions to this repo, to assume this identity. :thinking: 

I don't recall using `roboquat` for anything in `workspace-images`, yet. So the action might fail till we do something.

@securitymirco , or @aledbf , do you know what's needed for this to work? Alternatively, we could merge to main, run, and check for errors. 

Similar [action](https://github.com/gitpod-io/gitpod/blob/96bc40281b2507a5ba93c432d56dc8cb89c08ac7/.github/workflows/update-image-digest.yml).

Reply:
Looks like we need to add [this](https://github.com/gitpod-io/workspace-images/pull/1153/files#diff-3d61e19664dc79dcefdd581224a06ffd7d3615c8959c6ccb1ca7d818e855532eR71) secret?

Reply:
@kylos101 I could only guess, don't know for sure 😞

Reply:
Thanks for catching this, @kylos101 🙏. I suppose it needs to have the proper GitHub PAT to create the PR. I will rename the secret to something better digestible and would ask @securitymirco if he could create a @roboquat PAT with the `repo` permission and add it as a workspace secret under `ROBOQUAT_PR_TOKEN`? [[docs](https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#workarounds-to-trigger-further-workflow-runs)]. If you can, please also invite @roboquat to the repo so that it has access.

Reply:
@filiptronicek @kylos101 I've added roboquat with write permissions on this repository. the login credentials to roboquat can be found inside the 1Password PDE vault.